src/main/java/uk/gov/companieshouse/api/accounts/controller/DebtorsController.java

```diff
@@ -83,4 +87,72 @@ public ResponseEntity create(@Valid @RequestBody Debtors debtors,
         return responseEntity;
     }
 
+    @GetMapping
+    public ResponseEntity get(@PathVariable("companyAccountId") String companyAccountId,
+                              HttpServletRequest request) {
+
+        Transaction transaction = (Transaction) request
+                .getAttribute(AttributeName.TRANSACTION.getValue());
+
+        String accountingPoliciesId = debtorsService.generateID(companyAccountId);
+
+        ResponseEntity responseEntity;
+
+        try {
+            ResponseObject<Debtors> response = debtorsService
+                    .findById(accountingPoliciesId, request);
+
+            responseEntity = apiResponseMapper.mapGetResponse(response.getData(), request);
+
+        } catch (DataException de) {
+
+            final Map<String, Object> debugMap = new HashMap<>();
+            debugMap.put(TRANSACTION_ID, transaction.getId());
+            debugMap.put(COMPANY_ACCOUNT_ID, companyAccountId);
+            debugMap.put(MESSAGE, "Failed to retrieve debtors resource");
+            LOGGER.errorRequest(request, de, debugMap);
+            responseEntity = apiResponseMapper.map(de);
+        }
+
+        return responseEntity;
+    }
+
+    @PutMapping
+    public ResponseEntity update(@RequestBody @Valid Debtors debtors,
+                                 BindingResult bindingResult,
+                                 @PathVariable("companyAccountId") String companyAccountId,
+                                 HttpServletRequest request) {
+
+        SmallFull smallFull = (SmallFull) request.getAttribute(AttributeName.SMALLFULL.getValue());
+        if (smallFull.getLinks().get(SmallFullLinkType.DEBTORS_NOTE.getLink()) == null) {
+            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+        }
+
+        if (bindingResult.hasErrors()) {
+            Errors errors = errorMapper.mapBindingResultErrorsToErrorModel(bindingResult);
+            return new ResponseEntity<>(errors, HttpStatus.BAD_REQUEST);
+        }
+
+        Transaction transaction = (Transaction) request.getAttribute(AttributeName.TRANSACTION.getValue());
+
+        ResponseEntity responseEntity;
+
+        try {
+            ResponseObject<Debtors> response = debtorsService
+                    .update(debtors, transaction, companyAccountId, request);
+
+            responseEntity = apiResponseMapper
```

Comment:
We could `return` here and in line `153` to be consistent with lines `128` and lines `133`. So then, we could get rid of the reference variable **ResponseEntity responseEntity;**